packages/desktop/views/dashboard/developer/Developer.svelte

```diff
@@ -1,17 +1,17 @@
 <script lang="ts">
-    import { selectedAccount } from '@core/account'
-    import { localize } from '@core/i18n'
     import {
         FaucetRequestButton,
         MintNativeTokenButton,
         MintNftButton,
         CreateAliasButton,
-        Pane,
         RefreshTokenMetadataButton,
         Text,
         TestDeepLinkButton,
-    } from 'shared/components'
+    } from '@ui'
+    import { Pane } from '@components'
     import features from '@features/features'
+    import { selectedAccount } from '@core/account/stores'
```

Comment:
do we need to specify `/stores` now?

Reply:
Yes it helps out with circular dependency issues. David and I have started looking into a solution, hopefully we can try it soon